iset.mm

```diff
@@ -140474,6 +140474,79 @@ S C_ ( P ( ball ` D ) T ) ) $=
+          ( c2 cdiv co cfv cmin cabs clt wbr wa simpr cr ffvelrnd subcld abscld
+          wcel cc adantr rphalfcld rpred cc0 wceq cap wn simpll syl3anc ltnsymd
+          syl pm2.65da wb sseldd syl2an2r mpbird fveq2d subeq0bd abs00bd rpgt0d
+          apti crp eqbrtrd pm2.21dd wo rphalflt wi axltwlin mpd mpjaodan ) AEUA
+          UBUCZIFUDZCFUDZUEUCZUFUDZUGUHZWKEUGUHZWMAWLUIZWLWMAWLUJZWNWKWGAWKUKUO
+          ZWLAWJAWHWIABUPIFMRULZABUPCFMNULUMUNZUQAWGUKUOZWLAWGAEPURZUSZUQWNWKUT
+          WGUGWNWJWNWHWIAWHUPUOWLWQUQWNICFWNICVAZICVBUHZVCZWNXCWLWNWLXCWOUQWNXC
+          UIZWKWGXEAWPAWLXCVDZWRVGXEAWSXFXAVGXEAXCICUEUCUFUDDUGUHZWKWGUGUHXFWNX
+          CUJXEAXGXFTVGSVEVFVHAIUPUOWLCUPUOZXBXDVIABUPILRVJAXHWLABUPCLNVJUQICVQ
+          VKVLVMVNVOWNWGAWGVRUOWLWTUQVPVSVFVTAWMUJAWGEUGUHZWLWMWAZAEVRUOXIPEWBV
+          GAWSEUKUOWPXIXJWCXAAEPUSWRWGEWKWDVEWEWF $.
+      $}
+
+      $d A d e s z $.  $d B d e s z $.  $d F d e s z $.  $d K d e z $.
+      $d d e ph s z $.
+      $( Lemma for ~ cnplimccntop .  The reverse direction.  (Contributed by
+         Mario Carneiro and Jim Kingdon, 17-Nov-2023.) $)
+      cnplimclemr $p |- ( ph -> F e. ( ( J CnP K ) ` B ) ) $=
+        ( vz vd cfv co wcel cc clt crp ve vs cabs cmin ccom cxp cres cmopn ccnp
+        wf cv wbr wi wral wrex wa cap cdiv breq2 imbi2d rexralbidv climc sseldd
+        c2 wceq ellimc3ap simprd adantr rphalfcl adantl rspcdva ad5antr simpllr
+        mpbid ffvelrnd cnmetdval syl2anc wss simp-5r simp-4r w3a 3simpc simp1lr
+        eqid ovresd eqtrd simpr eqbrtrrd cnplimclemle eqbrtrd ralimdva reximdva
+        mpd exp31 ralrimiva cxmet cnxmet xmetres2 sylancr a1i metcnp2 mpbir2and
+        wb syl3anc crest metrest syl5eq oveq1d fveq1d eleqtrrd ) ADCUCUDUEZBBUF
+        UGZUHOZFUIPZOZCEFUIPZOADXOQZBRDUJZMUKZCXLPZNUKZSULZXSDOZCDOZXKPZUAUKZSU
+        LZUMZMBUNZNTUOZUATUNZJAYJUATAYFTQZUPZXSCUQULZXSCUDPUCOZYASULZUPZYCYDUDP
+        UCOZYFVDURPZSULZUMZMBUNZNTUOZYJYMYQYRUBUKZSULZUMZMBUNNTUOZUUCUBTYSUUDYS
+        VEZUUFUUANMTBUUHUUEYTYQUUDYSYRSUSUTVAAUUGUBTUNZYLAYDRQZUUIAYDDCVBPQZUUJ
+        UUIUPLAUBNMBCYDDJIABRCIKVCZVFVNVGVHYLYSTQAYFVIVJVKYMUUBYINTYMYATQZUPZUU
+        AYHMBUUNXSBQZUPZUUAYBYGUUPUUAUPZYBUPZYEYRYFSUURYCRQUUJYEYRVEUURBRXSDAXR
+        YLUUMUUOUUAYBJVLZUUNUUOUUAYBVMZVOUURBRCDUUSACBQZYLUUMUUOUUAYBKVLZVOYCYD
+        XKXKWDZVPVQUURBCYAYFDEFXSGHABRVRZYLUUMUUOUUAYBIVLZUUSUVBAUUKYLUUMUUOUUA
+        YBLVLAYLUUMUUOUUAYBVSYMUUMUUOUUAYBVTUUTUURYNYPWAYQYTUURYNYPWBUUPUUAYBYN
+        YPWCWMUURXTYOYASUURXTXSCXKPZYOUURXSCXKBUUTUVBWEUURXSRQCRQZUVFYOVEUURBRX
+        SUVEUUTVCAUVGYLUUMUUOUUAYBUULVLXSCXKUVCVPVQWFUUQYBWGWHWIWJWNWKWLWMWOAXL
+        BWPOQZXKRWPOQZUVAXQXRYKUPXCAUVIUVDUVHWQIXKBRWRWSUVIAWQWTKUANMXLXKCDXMFB
+        RXMWDZGXAXDXBACXPXNAEXMFUIAEFBXEPZXMHAUVIUVDUVKXMVEWQIXKXLFXMRBXLWDGUVJ
+        XFWSXGXHXIXJ $.
+    $}
+
+    $( A function is continuous at ` B ` iff its limit at ` B ` equals the $d x
+       A $. $d x B $. $d x F $. $d x K $. value of the function there.
```

Comment:
What is the meaning of the disjoint variables restrictions here?

Reply:
Oh copy paste mishap, thanks for noticing.

Should be fixed now.